src/frontend/sharedComponents/BGMapCard.tsx

```diff
@@ -34,23 +35,34 @@ interface BGMapCardProps {
   styleUrl: string;
   onPress?: (() => void) | null;
   isSelected: boolean;
+  bytesStored: number;
 }
 
 export const BGMapCard = ({
   mapTitle,
   style,
   isSelected,
   styleUrl,
+  bytesStored,
+  mapId,
 }: BGMapCardProps) => {
   const { formatMessage: t } = useIntl();
   const { position } = React.useContext(LocationContext);
-
+  const { navigate } = useNavigationFromRoot();
   return (
     <View
       style={[
         { borderColor: MEDIUM_GREY, borderWidth: 1, borderRadius: 2 },
         style,
       ]}
+      onTouchStart={() =>
```

Comment:
nit: weirdly this prop doesn't seem to be documented in the react native docs (although it works). generally, should use a [Touchable](https://reactnative.dev/docs/0.66/handling-touches#touchables) if a touch interaction is needed for something like this

Reply:
Good catch, this was originally a touchable, but we disabled the action, so i changed to a view, and didn't change it back